src/mscorlib/src/System/Exception.cs

```diff
@@ -83,8 +83,7 @@ protected Exception(SerializationInfo info, StreamingContext context)
             _stackTraceString = info.GetString("StackTraceString");
             _remoteStackTraceString = info.GetString("RemoteStackTraceString");
             _remoteStackIndex = info.GetInt32("RemoteStackIndex");
-
-            _exceptionMethodString = (String)(info.GetValue("ExceptionMethod", typeof(String)));
```

Comment:
As far as I can tell, the only consequence is that the Exception's TargetSite property might be null